media/filters/chunk_demuxer.h

```diff
@@ -130,6 +139,9 @@ class MEDIA_EXPORT ChunkDemuxerStream : public DemuxerStream {
   void UnmarkEndOfStream();
 
   // DemuxerStream methods.
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  std::string mime_type() const override;
```

Comment:
Please update media/filters/chunk_demuxer_unittest.cc .

Reply:
Added a TODO and a bug b/380001431 once media tests are up and running.